src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
+
+  let sum = 0;
+
+  men.forEach((m) => sum += m.died - m.born);
```

Comment:
Instead of `forEach`, use the most suitable array method - `reduce`.
Don’t shorten variables or function names. Check this [guide](https://mate-academy.github.io/style-guides/javascript.html#2311-dont-shorten-variables-or-functions-names).
```suggestion
  const sum = men.reduce((accumulator, man) => accumulator + man.died - man.born, 0);
```